### PR TITLE
docs: update macOS docs regarding outgoing connections

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

outgoing connections on macOS. Replacement PR for https://github.com/FirebaseExtended/flutterfire/pull/6376

## Related Issues

closes https://github.com/FirebaseExtended/flutterfire/issues/6356
## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: docs/installation/macos.mdx [hunk 1]
```diff
@@ -48,6 +48,12 @@ Specifically if your app is named 'MyApp', and are sharing your configuration be
 </dict>
 ```
 
+**Important** The next step is to enable `Outgoing Connections (Client)` on Xcode's `Runner.xcworkspace` as illustrated below:
```
Comment (Salakar) [inline docs/installation/macos.mdx:51]: Is this emulator specific?

Comment (russellwheatley) [reply docs/installation/macos.mdx:51]: I don't believe so actually. I was just following this [advice](https://github.com/FirebaseExtended/flutterfire/issues/6356#issuecomment-858560297) 😄. Can put under its own header if you wish?

File: docs/installation/macos.mdx [hunk 2]
```diff
@@ -32,6 +32,14 @@ Select the `GoogleService-Info.plist` file you downloaded, and ensure the "Copy
   caption={false}
 />
 
+## Enabling Outgoing Connections
+
+It is **important** you enable `Outgoing Connections (Client)` on Xcode's `Runner.xcworkspace` as illustrated below:
```
Comment (lesnitsky) [inline docs/installation/macos.mdx:37]: as per [this section](https://flutter.dev/desktop#setting-up-entitlements) it is recommended to edit `.entitlements` directly, not via UI. Maybe worth mentioning how to do this manually or just adding a link to this section

File: docs/installation/macos.mdx [hunk 3]
```diff
@@ -32,6 +32,14 @@ Select the `GoogleService-Info.plist` file you downloaded, and ensure the "Copy
...
+
+![image](https://user-images.githubusercontent.com/12681781/121333660-d1330500-c93a-11eb-8fb6-b1d803b9c7bb.png)
+
+Failure to enable will result in a network error. You will not b able to connect to services such as any Firebase Emulator.
```
Comment (lesnitsky) [inline docs/installation/macos.mdx:41]: will not b**e** able

**Review by lesnitsky — CHANGES_REQUESTED** [review]

**Review by lesnitsky — APPROVED** [review]

**Pull request merged**